components/Heroes/BenchmarkTable.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import style from './heroes.css';
+
+
+const metrics = [
+  'percentile',
+  'gold_per_min',
+  'xp_per_min',
+  'last_hits_per_min',
+  'kills_per_min',
+  'hero_damage_per_min',
+  'hero_healing_per_min',
+  'tower_damage_per_min',
+];
+const roundFn = (number) => Math.round(number * 100) / 100;
+const alignCenter = { textAlign: 'center' };
+
+const BenchmarkRow = ({ data, id, percentile }) => (
+  <TableRow selectable={false} className={style[`BenchmarkRow-${id % 2}`]}>
+    {metrics.map((metric, i) => {
+      if (metric === 'percentile') {
+        return (
+          <TableRowColumn key={i} style={alignCenter} className={style.BenchmarkColNo}>
+            {parseFloat(percentile) * 100}%
+          </TableRowColumn>
+        );
+      }
+
+      return (
+        <TableRowColumn key={i} style={alignCenter}>
+          {roundFn(parseFloat(data[metric][id].value))}
+        </TableRowColumn>
+      );
+    })}
+  </TableRow>
+);
+
+export default ({ data }) => (
+  <Table wrapperStyle={{ overflow: 'visible' }} className={style.BenchmarkTable}>
+    <TableHeader displaySelectAll={false} adjustForCheckbox={false}>
+      <TableRow>
+        <TableHeaderColumn tooltip="Percentile" style={alignCenter} className={style.BenchmarkColNo}>%</TableHeaderColumn>
```

Comment:
ok I will make new strings file in 'utility/language/en.js', do you think this is the appropriate way? or do we need use i18n/L10n?